internal/backend/remote-state/http/client.go

```diff
@@ -48,6 +49,8 @@ func (c *httpClient) httpRequest(method string, url *url.URL, data []byte, what
 		body = data
 	}
 
+	log.Printf("[DEBUG] Executing HTTP remote state request for: %q", what)
```

Comment:
The issue stated using debug logs since the user would want to know why they are getting non 200 response and if you look at the other backend clients especially oss it uses DEBUG.

Reply:
Do you mean only the logs within httpRequest method or all the DEBUG logs

Reply:
Thanks I understand I will update the log at line 52 to a trace log